.github/workflows/reusable-release.yaml

```
@@ -88,7 +88,7 @@ jobs:
           mkdir tmp    
 
       - name: GoReleaser
-        uses: goreleaser/goreleaser-action@v6
+        uses: goreleaser/goreleaser-action@90a3faa9d0182683851fbfa97ca1a2cb983bfca3 # v6.2.1
```

`goreleaser/goreleaser-action@v6` used version `v6.4.0` in the last test run:
https://github.com/aquasecurity/trivy/actions/runs/17317358101/job/49162716450#step:1:42
https://github.com/goreleaser/goreleaser-action/releases/tag/v6.4.0

```suggestion
        uses: goreleaser/goreleaser-action@e435ccd777264be153ace6237001ef4d979d3a7a # v6.4.0
```